provisioning/transport/http/devdoc/http_requirements.md

```diff
@@ -110,24 +106,22 @@ The `constructor` creates an Http transport object used to communicate with the
 
 **SRS_NODE_PROVISIONING_HTTP_18_022: [** operation status request polling shall be a GET operation sent to 'https://global.azure-devices-provisioning.net/{idScope}/registrations/{registrationId}/operations/{operationId}' **]**
 
-**SRS_NODE_PROVISIONING_HTTP_18_023: [** If the operation status request times out, `register` shall stop polling and call the `callback` with a `TimeoutError` error. **]**
+**SRS_NODE_PROVISIONING_HTTP_18_023: [** If the operation status request times out, `register` shall stop polling and call the `callback` with with the lower level error **]**
```

Comment:
>with the lower level [](start = 143, length = 20)

same thing here should be transport-agnostic I think (also duplicate 'with' word)